knox/knox.sh

```diff
@@ -128,22 +169,25 @@ function initialize_crontab() {
 function install() {
   local master_name
   master_name=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
-  if [[ "${master_name}" == $(hostname) ]]; then
-    [[ -z "${KNOX_GW_CONFIG_GCS}" ]] && err "Metadata knox-gw-config is not provided. knox-gw-config stores the configuration files for knox."
-
-    retry_command "apt-get update"
-    install_dependencies
-    update
-    enable_demo_ldap_for_dev_testing
-    get_config_parameters
-    replace_the_master_key
-    generate_or_replace_certificate
-    initialize_crontab
-
-    [[ -f "/lib/systemd/system/knoxldapdemo.service" ]] && systemctl restart knoxldapdemo
-    systemctl restart knox
+  if [[ "${master_name}" == $(hostname -s) ]]; then
+         [[ -z "${KNOX_GW_CONFIG_GCS}" ]] && err "Metadata knox-gw-config is not provided. knox-gw-config stores the configuration files for knox."
```

Comment:
if [[ -z "${KNOX_GW_CONFIG_GCS}" ]] ; then err "Metadata knox-gw-config is not provided. knox-gw-config stores the configuration files for knox." ; fi